proposals/NNNN-allows-navigation-apps-to-access-information-about-Wi-Fi-networks.md

```diff
@@ -7,7 +7,8 @@
 * Impacted Platforms: [Java Suite]
 
 ## Introduction
-This proposal requires all current navigation apps to include a new permission(android.permission.ACCESS_WIFI_STATE) to benefit from the issue fix as follows:
+This proposal requires all current navigation apps to include a new permission (android.permission.ACCESS_WIFI_STATE) to benefit from the issue fix as follows:
+

```

Comment:
```suggestion
```
Please remove this extra line.  I think it was included in my last suggestion by accident, sorry!